src/BloomBrowserUI/bookEdit/OverflowChecker/OverflowChecker.js

```diff
@@ -117,23 +97,22 @@ var OverflowChecker = (function () {
         else
             $page.addClass('pageOverflows');
     };
-
     OverflowChecker.CheckOnMinHeight = function (box) {
         var $box = $(box);
         var overflowy = $box.css("overflow-y");
         if (overflowy == 'hidden') {
             $box.css("min-height", $box.css("line-height"));
-        } else {
+        }
+        else {
             $box.css('min-height', '');
             var lineHeight = parseFloat($box.css("line-height"));
             var minHeight = parseFloat($box.css("min-height"));
-
             if (minHeight < lineHeight) {
                 $box.css("min-height", lineHeight + 0.01);
             }
         }
-
         $box.removeClass('Layout-Problem-Detected');
     };
     return OverflowChecker;
 })();
+//# sourceMappingURL=OverflowChecker.js.map
```

Comment:
The great majority of changes in this file appear to be just the result of a different typescript compiler. Since there's at least one other active PR in this area, it would be helpful to somehow suppress the spurious changes (likely to simplify merging).